<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-B4ND95H" failures="0" tests="2" name="com.testng.listeners.TestCase2" time="1.073" errors="1" timestamp="2024-07-13T22:33:08 IST" skipped="1">
  <testcase classname="com.testng.listeners.TestCase2" name="TestCase_2" time="0.001">
    <skipped/>
  </testcase> <!-- TestCase_2 -->
  <system-out/>
  <testcase classname="com.testng.listeners.TestCase2" name="TestCase_1" time="1.072">
    <error message="Method com.testng.listeners.TestCase2.TestCase_1() didn&#039;t finish within the time-out 1000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.testng.listeners.TestCase2.TestCase_1() didn't finish within the time-out 1000
at java.base@11.0.13/java.lang.Thread.sleep(Native Method)
at app//com.testng.listeners.TestCase2.TestCase_1(TestCase2.java:11)
at java.base@11.0.13/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@11.0.13/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base@11.0.13/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base@11.0.13/java.lang.reflect.Method.invoke(Method.java:566)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:44)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:72)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:10)
at java.base@11.0.13/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@11.0.13/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base@11.0.13/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base@11.0.13/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- TestCase_1 -->
  <system-out/>
</testsuite> <!-- com.testng.listeners.TestCase2 -->
